data_structures/stack/main.c

```diff
@@ -7,7 +7,7 @@ void peep();
 void update();
 int main()
 {
-    int n,a[100],top=0;
+    int n,a[100],top=-1;
     //function for pushing the element 
     void push()
```

Comment:
Can you remove the nested functions such as:
* push(...)
* pop()
* peep(...)  
Because nested functions are not a standard in C